app/config/locale/translations/esp.json

```
@@ -0,0 +1,231 @@
+{  
+     "settings.locale": "esp",
+     "settings.direction": "ltr",
+     "emails.sender": "Teamo %s",
+     "emails.verification.subject": "Konta Konfirmo",
+     "emails.verification.hello": "Saluton {{name}}",
+     "emails.verification.body": "Alklaku ĉi tiun ligon por kontroli vian retpoŝtan adreson.”,
```

```suggestion
     "emails.verification.body": "Alklaku ĉi tiun ligon por kontroli vian retpoŝtan adreson.",
```

Please fix the quotes. Looks like a misplaced quote somewhere which is causing the json syntax to break